data/ca/distr_data/readme.txt

```diff
@@ -1,8 +1,8 @@
-Kolibri OS 0.7.7.0+ night build, LiveCD-version.
-
-This is a bootable CD-Rom. To see the system in action, reboot and point to
-BIOS either in boot menu (if it can be called) or in settings booting from CD.
-
-Instructions for install to other information mediums and minimal system
-requirements can be found in file install.txt.
-Instructions for configuring can be found in file config.txt.
+Kolibri OS 0.7.7.0+ nightly build, Live CD version.
+
+This is a bootable CD-ROM. To run, reboot your computer and select the
+correct drive from the system settings or BIOS if required.
+
+The minimum system requirements and installation instructions can be found
+in 'install.txt'. Further details for configuring and customizing Kolibri
```

Comment:
Here and in other same places: Kolibri -> KolibriOS

Reply:
Thanks for the review and comment.
I preferred the more casual style of dropping 'OS' here,and avoids repetition... but happy to use only 'KolibriOS', if that's the community's wish.

Reply:
I absolutely understand your logic, we do use Kolibri and even kolibri informally. That said, README.TXT looks like a more formal text to me. Thank you for your contribution!

Reply:
Can you confirm if we should consistently use 'KolibriOS' vs 'Kolibri OS' (with a space) ...? Perhaps run a poll on the community forum? It would be good to start some notes on brand guidelines on the wiki.

Reply:
As a rule, we don't use a space, i.e. the right variant is KolibriOS (for example, check our [GSoC page](https://summerofcode.withgoogle.com/archive/2016/organizations/5678701068943360)).

> It would be good to start some notes on brand guidelines on the wiki.

As you see, we don't have any registered name or a trade mark. You are welcome to introduce informal guidelines. Let me know if you have any issues with registration on our [wiki](https://wiki.kolibrios.org).